### PR TITLE
Feat: Products page

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created Product page
- Listed Project
- Changed footer color to primary color
- added images for the product
 

Screenshot (202)

### CODE REVIEW

File: CSS/products.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+.free-badge{
+    background-color: var(--primary-color);
+    width: fit-content;
+    left: 0;
+}
+.free-badge > .badge{
+    margin-left: 0;
+}
+.free-badge span{
+    background-color: transparent !important;
```
Comment (srishti-maurya) [inline CSS/products.css:19]: suggestion: if possible you may avoid using `!important`

**Review by srishti-maurya — APPROVED** [review]
UI looks good 👍

**Review by saifudinbadani — COMMENTED** [review]
if possible decrease opacity of the badge little bit so the image behind is visible..other wise looks good to merge.

**Review by poornima-alugubelly — APPROVED** [review]
looks good!

**Pull request merged**